### DISCUSSION TITLE
ICE: found a bug with `rustc`

### DISCUSSION THREAD

**Discussion by clin1234** [dani-garcia/vaultwarden#5316] [General]
rust-lang/rust#134566

**Comment by BlackDex** [comment]
You can't install vaultwarden as a crate since we do not publish our code there as a crate.

**Reply by clin1234** [reply]
I installed straight from the Git repository, not through crates.io

**Reply by BlackDex** [reply]
I also see you are using rust 1.85, which Vaultwarden is not yet compatible with.

If you would use 1.83.0 (stable) and use run instead of install, what happens?
